site/src/main.rs

```diff
@@ -38,17 +42,13 @@ async fn main() {
                 "Loading complete; {} commits and {} artifacts",
                 commits, artifacts,
             );
-            eprintln!("View the results in a web browser at 'http://localhost:2346/compare.html'");
+            eprintln!("View the results in a web browser at 'http://localhost:{}/compare.html'", port);
```

Comment:
```suggestion
            eprintln!("View the results in a web browser at 'http://localhost:{port}/compare.html'");
```

Reply:
You can also use inline arguments, like this :) I wouldn't even mention it, but you'll need to reformat the code to pass CI anyway (by running `cargo fmt`), so you can make this change while you're at it :)